doc/topics/back-testing.md

```diff
@@ -16,6 +19,8 @@ backtest:
   - BTCUSDT
   
   account:
+    makerCommission: 15  # Commission rate of maker trade, in ‱, i.e., 1 means 0.01% 
+    takerCommission: 15  # Commission rate of taker trade, in ‱ i.e., 1 means 0.01% 
```

Comment:
this is out-dated and should be removed.

Reply:
But i still see commission fields in backtest result?

Reply:
we will remove it, because it is not consistent.

Reply:
Got it. Removed